azure-pipelines/TSAOptions.json

```diff
@@ -5,7 +5,7 @@
   "tsaStamp": "DevDiv",
   "tsaEnvironment": "PROD",
   "notificationAliases": [
-     "vsidemicrobuild@microsoft.com"
+     "andarno@microsoft.com"
```

Comment:
Does this have to be an individual or can we have a group email here?

Reply:
We _can_ have a group email there. But there's no DL that's focused on the owners of the vs-threading repo. If you have a better DL in mind, I'm happy to switch to it.